### DISCUSSION TITLE
Cake.OSSIndex

### DISCUSSION THREAD

**Discussion by pascalberger** [cake-build/cake#2992] [Extension requests]
Addin for querying [OSS Index](https://ossindex.net/public/restapi2)

**Comment by pascalberger** [comment]
Someone started implementation [here](https://github.com/TrekkingForCharity/Cake.OssIndex/)

**Comment by pascalberger** [comment]
Repo seems to be deleted now

**Comment by AdmiringWorm** [comment]
Seems like the url to the rest api have changed to: https://ossindex.net/rest
